src/main/resources/reference.conf

```diff
@@ -84,6 +84,11 @@ lorre {
   GETResponseEntityTimeout: 90 s
   POSTResponseEntityTimeout: 1 s
 
+  # defines custom updates to be run at specific epochs (i.e. upon reaching a given level)
+  chain-epoch-updates {
+    accounts-refresh-epochs: []
```

Comment:
Can we also accept a SQL pattern which we can use to filter accounts which are refreshed, e.g. `tz1%` or `KT19EnJqbfWuQ6oWAZZWLjBzibNMqMFGPcW2`? This will make this new functionality a lot more useful.

Reply:
Absolutely... I suggest I might work on that on next PR